Rakefile

```diff
@@ -42,7 +42,8 @@ namespace :update do
     .binaryTarget(
       name: "swiftformat",
       url: "https://github.com/calda/SwiftFormat-nightly/releases/download/#{latest_version_number}/SwiftFormat.artifactbundle.zip",
-      checksum: "#{checksum.strip}"),
+      checksum: "#{checksum.strip}"
+    ),
```

Comment:
While we're here, update the `update:swiftformat` rake task to output correctly-formatted code